docs/tutorials/hyp3_insar_stack_for_ts_analysis.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.    project_home = Path.cwd()
</pre><p>we don't use <code>project_home</code> again, so you can just define  <code>work_dir = Path.cwd() / project_name</code> and drop this variable </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ASFHyP3/hyp3-docs/pull/213/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>